File: 26. Remove Duplicates from Sorted Array/solution.cpp
class Solution {
public:
  int removeDuplicates(vector<int>& nums) {
    int unique = 1;
    for (int index = 1; index < nums.size(); index++){
      if (nums[index] != nums[index - 1]){
        nums[unique] = nums[index];
        unique += 1;
      }
    }
    return unique;
  }
};

File: 26. Remove Duplicates from Sorted Array/solution.py
class Solution(object):
  def removeDuplicates(self, nums):
    """
    :type nums: List[int]
    :rtype: int
    """
    unique = 1
    for index in range(1, len(nums)):
      if nums[index] != nums[index - 1]:
        nums[unique] = nums[index]
        unique += 1
    return unique